### ISSUE TITLE
NPM package doesn't contain all files

### ISSUE THREAD

**Issue by shiroari** [jsdf/find-index#5]
Package is missing new files: `findLastIndex.js` and `findIndex.js`. Please check your `package.json`. I think it would be better if you drop `files` property in `package.json` and use `.npmignore`.

```
[ERROR] Error: Cannot find module './findIndex'
[ERROR]     at Function.Module._resolveFilename (module.js:338:15)
[ERROR]     at Function.Module._load (module.js:280:25)
[ERROR]     at Module.require (module.js:364:17)
[ERROR]     at require (module.js:380:17)
```

**Comment by jsdf** [comment]
d'oh.

**Comment by jsdf** [comment]
Published a new version v1.0.1 and deprecated v1.0.0
